crates/serpen/tests/fixtures/comprehensive_ast_rewrite/models/base.py

```
@@ -0,0 +1,127 @@
+"""
+Base model functionality with relative import conflicts
+"""
+
+from typing import Any, Dict
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)